widgetastic_manageiq.py

```diff
@@ -878,3 +875,57 @@ def last_page(self):
     @property
     def items_amount(self):
         return self.paginator.page_info()[1]
+
+
+class AlertEmail(Widget):
+
+    ROOT = "//div[@id='edit_email_div']"
+    ADD_BUTTON = ".//div[@title='Add']"
+    RECIPIENTS = "./div[@id='edit_to_email_div']//a"
+    EMAIL_LINK = ".//a[text()='{}']"
```

Comment:
Thinking about this, This might as well get converted to a View-Widget. These locators would become widgets set on the class and the init would call out to View instead of Widget (with the same signature). That sohuld then work.